apollo-portal/src/main/resources/static/index.html

```diff
@@ -36,87 +37,189 @@
+                <i class="iconfont icon-liulan" style="color: #666"></i>
+                <h5>{{'Index.RecentlyViewedItems' | translate }}</h5>
+            </div>
+        </div>
+        <div class="main-table">
+            <div class="create-app-list" ng-show="whichContent==='1'">
+                <div style="width: 180px;margin-left: 30px" ng-click="goToCreateAppPage()"
+                     ng-if="hasCreateApplicationPermission">
+                    <div href="#" class="thumbnail create-btn hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img src="img/plus-white.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.CreateProject' | translate }}</h5></div>
                     </div>
                 </div>
-                <div class="app-panel col-md-2 text-center" ng-repeat="app in createdApps"
-                    ng-click="goToAppHomePage(app.appId)">
-                    <div href="#" class="thumbnail hover cursor-pointer">
-                        <h4 ng-bind="app.appId"></h4>
-                        <h5 ng-bind="app.name"></h5>
-                    </div>
+                <div class="display-table" ng-show="createdApps.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 15%">{{'Common.AppId' | translate }}</th>
+                            <th style="width: 25%">{{'Common.AppName' | translate }}</th>
+                            <th style="width: 20%">{{'Common.Department' | translate }}</th>
+                            <th style="width: 20%">{{'Common.AppOwner' | translate }}</th>
+                            <th style="width: 20%">{{'Common.Email' | translate }}</th>
+                        </tr>
+                        <tr ng-repeat="app in createdApps" ng-click="goToAppHomePage(app.appId)" href="#" class="hover cursor-pointer">
+                            <td style="width: 15%">{{ app.appId }}</td>
+                            <td style="width: 25%">{{ app.name }}</td>
+                            <td style="width: 20%">{{ app.orgName + '(' + app.orgId + ')' }}</td>
+                            <td style="width: 20%">{{ app.ownerName }}</td>
+                            <td style="width: 20%">{{ app.ownerEmail }}</td>
+                        </tr>
+                    </table>
                 </div>
-                <div class="app-panel col-md-2 text-center" ng-show="hasMoreCreatedApps"
-                    ng-click="getUserCreatedApps()">
-                    <div href="#" class="thumbnail hover cursor-pointer">
-                        <img class="more-img" src="img/more.png" />
-                        <h5>{{'Index.LoadMore' | translate }}</h5>
+                <div ng-show="!hasMoreCreatedApps" style="height: 15px"></div>
+                <div style="width: 180px;margin-left: 30px;margin-top: 20px" ng-show="hasMoreCreatedApps"
```

Comment:
here is a nice to have suggestion, since we are using table to present the items, it will be nice to add a [pagination](https://angular-ui.github.io/bootstrap/#!#pagination) control here